tests/test_experiment.py

```diff
@@ -446,3 +446,51 @@ def test_firefox_ios_app_id_propagation():
     assert "my_cool_app" in enrollments_sql
 
     sql_lint(metrics_sql)
+
+
+def test_exposure_query():
+    exp = Experiment("slug", "2019-01-01", 8, app_id="my_cool_app")
+
+    tl = TimeLimits.for_ts(
+        first_enrollment_date="2019-01-01",
+        last_date_full_data="2019-03-01",
+        time_series_period="weekly",
+        num_dates_enrollment=8,
+    )
+
+    enrollment_sql = exp.build_enrollments_query(
+        time_limits=tl,
+        enrollments_query_type="glean-event",
+    )
+
+    sql_lint(enrollment_sql)
+
+    assert "exposure" in enrollment_sql
+
+
+def test_exposure_metric_query():
+    exp = Experiment("slug", "2019-01-01", 8, app_id="my_cool_app")
+
+    tl = TimeLimits.for_ts(
+        first_enrollment_date="2019-01-01",
+        last_date_full_data="2019-03-01",
+        time_series_period="weekly",
+        num_dates_enrollment=8,
+    )
+
+    enrollment_sql = exp.build_enrollments_query(
+        time_limits=tl,
+        enrollments_query_type="glean-event",
+        exposure_metric=Metric(
+            name="exposure",
+            data_source=mozanalysis.metrics.fenix.baseline,
+            select_expr="metrics.counter.events_total_uri_count > 0",
+            friendly_name="URI visited exposure",
+            description="Exposed when URI visited",
+        ),
```

Comment:
I'm wondering if it makes sense to have an `ExposureMetric` type here? By default `Metric`s are assumed to have a `select_expr` that is used in a `GROUP BY`. But in the current implementation the `select_expr` is used as a condition of when an exposure happened.

Reply:
If they have different semantics I agree the types should be different, but could they have the same semantics? It seems like we could group by client_id and branch (sort of unnecessarily) and have the same aggregation semantics. Is that a good idea?

Reply:
Kind of. However, I think a separate type makes sense here to make the distinction between normal metrics and exposure metrics clearer. Otherwise, it might not be entirely clear that most of the pre-defined metrics, like `metrics.desktop.baseline_ping_count`, don't work correctly here although they have the right type. `select_expr` still expects some kind of condition to determine when an exposure happened vs. just an aggregation of metric values.

Reply:
Sounds good! :bikeshed: but maybe this isn't a `Metric` at all then; how about a `ExposureSignal`? It would be satisfying if we could implement the Nimbus exposure event in terms of whatever the abstraction we use for these events is.

Reply:
Calling it `ExposureSignal` sounds good 👍